### PR TITLE
multi browser testing implementation

### PR DESCRIPTION
**Pull request by TetianaFilatova**

### CODE REVIEW

File: src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
-@Listeners(TestNgListeners.class)
 public class BaseTestRunnerUI {
 
     protected static TestValueProvider valueProvider;
     protected WebDriver driver;
     protected SoftAssert softAssert = new SoftAssert();
+    Browsers browsers = new Browsers();
+
+    private void checkErrorPage(String browser) throws InterruptedException {
+        try {
```
Comment (Maksym637) [inline src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java:28]: Do we need throw an InterruptedException ?

**Review by janestadnytska — APPROVED** [review]

File: src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java [hunk 2]
```diff
@@ -28,31 +41,21 @@ public void initTestValueProvider() {
...
-        driver.findElement(By.id("proceed-link")).click();
+        checkErrorPage(browser);
     }
 
+    @Parameters("browser")
     @AfterClass
     public void closeDriver() {
         if (driver != null) {
             driver.close();
         }
```
Comment (Maksym637) [inline src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java:59]: change to `driver.quit()`, as after tests we need close window and also kill process in Task Manager. 
`driver.quit()` is more suitable in this case

File: src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java (hunk 1 above)
Comment (TetianaFilatova) [reply src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java]: fix

File: src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java (hunk 2 above)
Comment (TetianaFilatova) [reply src/test/java/org/ssu/edu/teachua/utils/runners/BaseTestRunnerUI.java]: done

**Review by Maksym637 — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**